### PR TITLE
chore: Update CODEOWNERS

### PR DESCRIPTION
**Pull request by VeeruPrudhvi**
Update the codeowners file for the default reviewers list.

### CODE REVIEW

**Review by cshilwant — CHANGES_REQUESTED** [review]
@VeeruPrudhvi `processor-sdk-doc` uses `commit-check-action` github action that follows [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for commit message.

Hence, we see the Commit Check GitHub workflow failing.
Checkout https://github.com/TexasInstruments/processor-sdk-doc/actions/runs/12391426830?pr=141 for more details

In your case, the commit message can be similar to https://github.com/TexasInstruments/processor-sdk-doc/pull/78/commits/56ca25b1189fe22d2b639e83ce72ebbeb7a77851 which is an accepted format & ensures that the workflow passes.

Also, ensure you Sign-off each commit by adding a `-s` flag with `git commit` - https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--s

**Review by jsuhaas22 — DISMISSED** [review]
LGTM

**Review by jeevantelukula — APPROVED** [review]

**Review by DhruvaG2000 — CHANGES_REQUESTED** [review]
I see a lot of unintended changes here due to some git mishap probably. Please no merge commits and no REVERT merge commits either. Just git pull --rebase on master and then ``force push`` to the PR branch.

File: .github/CODEOWNERS
Comment (cshilwant) [inline .github/CODEOWNERS:1]: Please plan to address https://github.com/TexasInstruments/processor-sdk-doc/pull/141#pullrequestreview-2513688632

**Review by cshilwant — CHANGES_REQUESTED** [review]

**Review by cshilwant — CHANGES_REQUESTED** [review]
Please plan to address https://github.com/TexasInstruments/processor-sdk-doc/pull/141#pullrequestreview-2513688632

**Pull request closed without merging**